.github/travis/success.sh

```diff
@@ -21,7 +21,11 @@ git config --local user.name "Deployment Bot"
 git config --local user.email "verible-dev@googlegroups.com"
 
 case $MODE in
-compile-n-test)
+test)
```

Comment:
Just wondering: is there a concern with `test` latency that is keeping this separate/distinct from `compile`?